src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,43 @@
+.clock {
+  position: relative;
+  width: $container_size;
+  height: $container_size;
+  box-sizing: border-box;
+  border: 1px solid black;
+}
+
+.clock_hour {
+  position: absolute;
+  width: $arrows_width;
+  height: $arrows_hour-height;
+  top: $center - $arrows_hour-height;
+  left: $indent;
+  background-color: yellow;
+  border: 1px solid black;
+  border-radius: $border_rad-arrows;
+  animation: 60s ease-in 0s infinite normal move;
+}
+
+.clock_minute {
+  position: absolute;
+  width: $arrows_width;
+  height: $arrows_minute-height;
+  top: $center - $arrows_minute-height;
+  left: $indent;
+  background-color: royalblue;
+  border: 1px solid black;
+  border-radius: $border_rad-arrows;
+  animation: 1s linear 0s infinite normal move;
+}
```

Comment:
Follow DRY (Don't Repeat Yourself )principe. 
There are a lot of repeatable styles according to `.clock_hour` selector. So use `placeholders`/`mixins`, common selector with `modifier` (BEM) or list selectors with the same rules